test/sideinput-e2e/testdata/map-sideinput-pipeline.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: sideinput-pipeline
+spec:
+  sideInputs:
+    - name: myticker
+      container:
+        image: "quay.io/numaio/numaproj-contrib/e2e-map-sideinput-example:v0.0.1"
+        imagePullPolicy: Always
+      trigger:
+        schedule: "@every 5s"
+  vertices:
+    - name: in
+      source:
+        http: {}
```

Comment:
Can we use a [generator source](https://numaflow.numaproj.io/user-guide/sources/generator/) instead of HTTP? It simplifies the test.

Reply:
No we can't use generator source because it generated random data, so in that case we will not able to predict what we need to look in redis to validate the result.

Reply:
Oh I see, you updated the UDF implementation to actually use the input messages. sounds good.

Reply:
sorry, did you update udf? According to https://github.com/numaproj-contrib/e2e-tests-go/pull/1/files#diff-ff48b126ed8c8f680115c9c4a2f7335e6bc462d6654abffb4e5253856ba39790R35 , the input message is not used.

Reply:
I missed the use of datum. Looks good to me